docs/doxygen.cfg.in

```diff
@@ -1013,7 +1013,7 @@ GENERATE_HTML          = YES
 # The default directory is: html.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_OUTPUT            = html
+HTML_OUTPUT            = 
```

Comment:
Inside `install/ ` I see `install/docs/html/html/`
repetition of html/ folder twice, is that expected?
I tried removing the default location, I think that doesn't seem to work, any ideas @DavidTruby ?

Reply:
Not sure about this one, but I think clang's docs also end up in the same place? If so that's fine.